qiita_db/util.py

```diff
@@ -2334,7 +2334,7 @@ def send_email(to, subject, body):
     msg = MIMEMultipart()
     msg['From'] = qiita_config.smtp_email
     msg['To'] = to
-    msg['Subject'] = subject
+    msg['Subject'] = subject.replace('\n', '')
```

Comment:
subject.strip() or subject.rstrip() w/out parameters will also strip the newline from strings and is more concise. It also has the side benefit of removing '\r\n' for cases when you're reading in from a Windows text file that has carriage return as well as line-feed.

Reply:
strip() removes leading whitespace from both the left and right sides of a string, while rstrip() only cleans up the end of the string. It's perhaps the more conservative choice to use for this situation.